.github/workflows/update-load-generator-image.yml

```
@@ -0,0 +1,56 @@
+# This workflow uses GHA to build a Java project with Gradle 'load-generator', and build and push multi-arch docker image of this project to the Amazon ECR.
+
+name: Update Load-Generator Image
+
+# Should execute only one workflow run at a time
+concurrency:
+  group: build-and-push-load-generator-image
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ['terraform']
+
+    # List of paths that should be considered when triggering the workflow
+    paths: ['load-generator/**']
+
+permissions:
+  id-token: write
+  contents: write
```

```suggestion
  contents: read
```

I don't think this write permissions to the repository does it?

---

Ok. Thanks. I will update this.

---

Thanks I have updated this.